projects/canopy/src/lib/promo-card/promo-card.stories.ts

```diff
@@ -0,0 +1,112 @@
+import { select, withKnobs } from '@storybook/addon-knobs';
+import { moduleMetadata } from '@storybook/angular';
+
+import { notes } from './promo-card.notes';
+import { LgPromoCardModule } from './promo-card.module';
+import { LgGridModule } from '../grid';
+import { LgHeadingModule } from '../heading';
+import { LgButtonModule } from '../button';
+import { LgSeparatorModule } from '../separator';
+import { LgSpacingModule } from '../spacing';
+
+const cardListConfig = {
```

Comment:
Not sure what others think but we could hook this data up to knobs like we do for the table..
https://deploy-preview-219--legal-and-general-canopy.netlify.app/?path=/story/components-table--standard
It gives people the chance to try out different data in the component, useful for testing.

Reply:
Thus has been discussed and will be left as it is for the time being.

Reply:
I'm happy either way with this one, with a small inclination not to do it - for the Promo Cards it's a little different, as you're only supposed to have two or three cards in the list, and the copy and images have to meet the design requirements, so it's kinda of limited, whereas the table has a lot more possibilities (unlimited rows, empty cells etc). The table is also often populated with JSON data in real world use - whereas you could argue Promo Cards probably wouldn't be (although ironically their first outing in MA will be) 🙂 
Having said that, I'd be fine if it was done.